eclipse-theia/theia
# feat(ai): add hovers for agents and variables

#### What it does

* Highlights the parsed chat request agent and variable parts
* Adds hover information for the agent and variable parts
* Adds hover information for the agent label in response headers

![image](https://github.com/user-attachments/assets/4f25464d-c13f-4267-83b2-f15956e33938)

#### How to test

Use variables and agent references in your requests and check whether they are correctly rendered.
Check if they have hover information.
Make sure generic markdown formatting still works.

#### Follow-ups

None

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
